tools/startup/setup.sh

```diff
@@ -0,0 +1,114 @@
+
+#!/bin/bash
+
+# Setup script. Please connect to wifi before running (see TODO below).
+# Note: Because of the FIXME, you'll have to run it, run "source ~/.bashrc", and run it again.
+
+# Make us the owners of the files in data
+sudo chown -R $USER /media/ubuntu/data
+cd /media/$USER/data
+
+if [ ! -d downloads ]; then
+  mkdir downloads
+fi
+
+# Uninstall unneeded packages
+sudo apt-get purge -y thunderbird*
+sudo apt-get purge -y firefox*
+sudo apt-get purge -y libreoffice*
+sudo apt-get purge -y gnome-sudoku
+sudo apt-get purge -y gnome-mines
+sudo apt-get purge -y gnome-mahjongg
+sudo apt-get purge -y gnome-calendar
+sudo apt-get purge -y *gnome-todo*
+sudo apt-get purge -y transmission*
+sudo apt-get purge -y deja-dup
+sudo apt-get purge -y aisleriot
+sudo apt-get purge -y *mozc*
+sudo apt-get purge -y *rhythmbox* gir1.2-rb-3.0:amd64
+sudo apt-get purge -y ubuntu-web-launchers
+
+# Copy wifi credentials
+mkdir -p wifis
+sudo cp /etc/NetworkManager/system-connections/* wifis/
+sudo cp wifis/* /etc/NetworkManager/system-connections/
+sudo systemctl restart NetworkManager
+
+# Download apt installs if needed
+# TODO: Figure out how to download packages from a predefined list, instead of "latest". This list will be generated from "latest" at some point in time.
```

Comment:
@oferb is [this](https://www.pcsuggest.com/apt-generate-package-download-list/) something that you had in mind?

Reply:
Something along those lines. I'd like to get a reproducible setup.